client/post-editor/editor-ground-control/style.scss

```diff
@@ -38,15 +47,36 @@
 	margin: 0 4px 8px;
 }
 
-.editor-ground-control__publish-button {
+.editor-ground-control .editor-publish-button {
```

Comment:
`editor-publish-button` is just a `<Button>` design-wise, with some style tweaks depending on where on the page it's rendered. I don't know if this was discussed before, but the general approach is to put these "container-dependant tweaks" in the container's stylesheet (in this case, `editor-ground-control` and `editor-mobile-navigation`).